### DISCUSSION TITLE
저 혹시  Resources.Load 모바일에서 작동되나요?

### DISCUSSION THREAD

**Discussion by zeze103** [naverz/zepeto-studio-kor#1096] [5. Q&A]
월드 1.5.0 버전에서 개선된 기능에 

Windows 환경에서 빌드 된 월드를 모바일에서 실행 시 Resource.Load가 정상적으로 동작하지 않는 문제 수정

이라고 되어 있길래 테스트 해봤는데 여전히 안 불러지던데, 혹시 되는 분 계시나요?
아니면, Resource.Load 로 불러올수는 오브젝트가 한정되어 있는건가요?

현제 버전은 1.5.1버전을 사용중이고, 테스트도 일반 PC에서는 잘되는데 QR 생성또는 테스트 링크 생성 후에 모바일에서 실행하면 안되었습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 모바일에서만 동작하지 않는다면 ZEPETO 앱 버전이 3.14.0 이상인지 확인 부탁 드립니다.
월드 버전과 제페토 앱 버전이 같이 버전업되어야 정상적으로 동작합니다.

**Reply by zeze103** [reply]
앱 버전은 3.14.2 버전이며, 앱 업데이트 일자는 7월 14일에 업데이트 되어 있습니다.

**Reply by Zepeto-TechEvangelist** [reply]
일반적인 리소스인 이미지나 텍스트는 정상적으로 불러와져야 하는데 파악이 필요해 보입니다.
관련 코드나 스크린샷 첨부 부탁 드립니다.

**Reply by zeze103** [reply]
늦게 확인하여 죄송합니다;;

다시 코드 수정하여 테스트해본 결과 다른 코드와 연결된 코드에서 나온 오류가 있어 생성이 안되었던 건게 확인되었습니다;
멀티에서  Resource.Load는 정상적으로 작동되었습니다;;
